code/modules/paperwork/pen.dm

```diff
@@ -338,3 +338,17 @@
 			P.contact_poison_poisoner = user.name
 			add_attack_logs(user, P, "Poison pen'ed")
 			to_chat(user, "<span class='warning'>You apply the poison to [P].</span>")
+
+// MARK: CHAMELEON PEN
+/obj/item/pen/chameleon
+	var/forge_name = null
```

Comment:
```suggestion
	var/forge_name
```

Reply:
Can you please explain, why it's not good to set it to null. And what this var will be initialized with?